dashboard/app/models/user.rb

```diff
@@ -201,6 +201,7 @@ class User < ActiveRecord::Base
     class_name: 'Pd::Application::ApplicationBase',
     dependent: :destroy
 
+  has_many :sign_ins
```

Comment:
since we are just adding this now, will we get into issues of everyone seeing the race interstitial because their first sign in will be when we merge this code? Or has it been running already we just haven't accessed it explicitly?

Reply:
Yeah exactly re: running already and not explicitly accessed, we've been logging to this table since early 2017 (289 million rows 😬 ). So, it's possible that a user would have no entries in this table (eg, account created 2016, hasn't signed in since), which I think I've accounted for 🤞.

Reply:
oh neat!